src/java.base/unix/native/libnet/PlainDatagramSocketImpl.c

```diff
@@ -1837,6 +1837,9 @@ static void mcast_join_leave(JNIEnv *env, jobject this,
     jint fd;
     jint family;
     jint ipv6_join_leave;
+#ifdef __APPLE__
+    int res;
+#endif
 
```

Comment:
res will need to be declared unconditionally, no? ( since it is used on all platforms )

Reply:
yes ... good spot 👍  ..... as testing has been focused on macos, this error didn't show